support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -147,14 +147,14 @@ Evolution:
   # This is the smallest interval we'd need to observe time step/constraints. If
   # you need it smaller you can edit it, but make sure to change the slab
   # intervals in the EventsAndTriggers
-  InitialSlabSize: 0.5
+  InitialSlabSize: 0.1
   StepChoosers:
     - LimitIncrease:
         Factor: 2
     - PreventRapidIncrease
     - ErrorControl:
-        AbsoluteTolerance: 1e-8
-        RelativeTolerance: 1e-6
+        AbsoluteTolerance: 1e-10
+        RelativeTolerance: 1e-8
```

Comment:
I adjusted the excision radii in the evolution by trial and error to avoid incoming char speeds when starting the evolution. I found that not excising as far in as the initial data did worked better in my runs.